gilda/grounder.py

```diff
@@ -87,19 +87,25 @@ def _generate_lookups(self, raw_str):
                     ', '.join(lookups))
         return lookups
 
-    def ground(self, raw_str):
+    def ground(self, raw_str, context=None, sort=False):
```

Comment:
maybe `sort=True` would be better? I put it as `False` to maintain backwards compatibility.